build/autotools/m4/ac_compile_check_sizeof.m4

```diff
@@ -1,9 +1,10 @@
 AC_DEFUN([AC_COMPILE_CHECK_SIZEOF],
-[
+[changequote(<<, >>)dnl
```

Comment:
Can you explain why we need changequote at all?

I think that we do *not* need changequote. What's actually needed is to fix line 12. Instead of this:

    #include &lt;sys/types.h&gt;

It should simply be:

    #include <sys/types.h>

The changequote seems like a failed attempt to accomplish this? In any case it doesn't seem necessary to me, I think we can use "<" and ">" directly in this file.

Let me know if that seems correct to you.

Reply:
From the m4 documentation, changequote is recommanded to avoid expansion in define() and redefine a macro the safe way. But you're right, in this particular issue, we don't need it at all.

Changequote is not concerned about the `include`, but we can safely edit it. Seems completely correct to me.